heat/core/tests/test_types.py

```diff
@@ -189,3 +189,20 @@ def test_type_promotions(self):
             ht.promote_types(1, '?')
         with self.assertRaises(TypeError):
             ht.promote_types(ht.float32, 'hello world')
+
+class TestTypeConversion(unittest.TestCase):
+    def test_finfo(self):
+        info32 = ht.types.finfo(ht.float32)
```

Comment:
It would be good if finfo was accessible already on module level to be consistent with numpy, i.e.

`ht.finfo(ht.float32)`

Just needs a qualified import in one of the __init__.py. The implementation is in the correct file though.

Reply:
Indeed a good idea, I added finfo as  import to the init.py to cover this.